streaming/src/test/scala/org/apache/spark/streaming/CheckpointSuite.scala

```diff
@@ -640,12 +640,14 @@ class CheckpointSuite extends TestSuiteBase with DStreamCheckpointTester
         val fileStream = ssc.textFileStream(testDir.toString)
         // Make value 3 take a large time to process, to ensure that the driver
         // shuts down in the middle of processing the 3rd batch
-        CheckpointSuite.batchThreeShouldBlockIndefinitely = true
+        CheckpointSuite.batchThreeShouldBlockALongTime = true
         val mappedStream = fileStream.map(s => {
```

Comment:
Yeah feel free to also make that style change while you're here. The ( and { are redundant and just needs a {